packages/@ionic/cli/src/lib/integrations/capacitor/index.ts

```diff
@@ -195,9 +195,16 @@ export class Integration extends BaseIntegration<ProjectIntegration> {
     if (!output) {
       debug('Could not get config from Capacitor CLI (probably old version)');
       return;
+    } else {
+      try {
+        // Capacitor 1 returns the `command not found` error in stdout instead of stderror like in Capacitor 2
+        // This ensures that the output from the command is valid JSON to account for this
+        return JSON.parse(output);
+      } catch(e) {
+        debug('Could not get config from Capacitor CLI (probably old version)', e);
```

Comment:
This could potentially silence legitimate JSON errors which would've previously halted the execution.
We could check if the error is an instance of [SyntaxError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions) if we want to still properly bail out when a syntax error is found in the JSON, and perhaps check the running version of Capacitor instead to purposefully swallow Cap1 errors?

Reply:
I think that makes sense, but would there be a difference in behavior? Right now if we can't get the Capacitor config from the CLI then we fallback to trying to read a `capacitor.config.json`, and I think that would still be the same behavior with the changes you proposed.

Reply:
Sounds good!